js/viewer/annotation/annotation-group.js

```diff
@@ -48,6 +48,20 @@ function AnnotationGroup(id, anatomy, description, parent){
             case "LINE":
                 annotation = new Line(attrs);
                 break;
+            case "ARROWLINE":
+                group = document.getElementById(this.id);
```

Comment:
Comment the purpose of what you're doing here.

Reply:
Added the required comments.